src/components/JobApplicationDialog.jsx

```diff
@@ -194,13 +193,13 @@ export default function JobApplicationDialog({jobApplication, handleClose, open,
                                 setJobUrl(e.target.value);
                             }}
                             defaultValue={jobApplication.jobUrl}
-
                         />
 
                         <DialogContent />
                             <FormControl sx={{ minWidth: 120 }} variant="standard" size="small">
                                 <InputLabel id="status-label">Status</InputLabel>
                                 <Select
+                                    disabled={isNew}
```

Comment:
um....now I see what you meant by setting it default do `open`. I think this makes over 90% of the use case. I like that.